<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudent</class>
 <widget class="QWidget" name="AddStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Student Info&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstname_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>First Name </string>
   </property>
  </widget>
  <widget class="QLabel" name="secondname_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>Second Name </string>
   </property>
  </widget>
  <widget class="QLabel" name="age_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstname_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondname_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="grade_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="grade_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>Grade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #0077b3; /* Medium blue background for buttons */
    color: #FFFFFF; /* White text */
    border: none; /* No border */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 16px; /* Adjust padding as needed */
}

QPushButton:hover {
    background-color: #4da6ff; /* Lighter blue on hover */
}

QPushButton:pressed {
    background-color: #005c99; /* Darker blue when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #0077b3; /* Medium blue background for buttons */
    color: #FFFFFF; /* White text */
    border: none; /* No border */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 16px; /* Adjust padding as needed */
}

QPushButton:hover {
    background-color: #4da6ff; /* Lighter blue on hover */
}

QPushButton:pressed {
    background-color: #005c99; /* Darker blue when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
